packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/representation/services/RepresentationMetadataSearchService.java

```diff
@@ -50,8 +50,8 @@ public Optional<RepresentationMetadata> findByRepresentationId(String representa
 
     private Optional<RepresentationMetadata> findTransientRepresentationById(String representationId) {
         Optional<RepresentationMetadata> representationMetadata = Optional.empty();
-        if (representationId.startsWith(ExplorerDescriptionProvider.REPRESENTATION_ID)) {
-            representationMetadata = Optional.of(new RepresentationMetadata(ExplorerDescriptionProvider.REPRESENTATION_ID, Tree.KIND, ExplorerDescriptionProvider.REPRESENTATION_NAME, ExplorerDescriptionProvider.DESCRIPTION_ID));
+        if (representationId.startsWith(ExplorerDescriptionProvider.PREFIX)) {
```

Comment:
This is a duplicate of RepresentationApplicationService1findRepresentationMetadataById, it was a mistake on my side.